pkg/cmd/roachtest/tests/ledger.go

```diff
@@ -43,9 +43,14 @@ func registerLedger(r registry.Registry) {
 				concurrency := roachtestutil.IfLocal(c, "", " --concurrency="+fmt.Sprint(nodes*32))
 				duration := " --duration=" + roachtestutil.IfLocal(c, "10s", "10m")
 
+				labels := map[string]string{
```

Comment:
nit: some tests we create a map literal like here to initialize the labels vs. some tests we do `labels := make(map[string]string)`. Any reason for one over the other?

Reply:
This was a miss from my side. Thanks for pointing out. I have fixed the case where `labels := make(map[string]string)` was used